Origin: commit 18ca43507b351f339ff23062541ee8d58e813a53
Author: Florian Forster <ff@octo.it>
Bug: https://github.com/octo/liboping/issues/38
Bug-Debian: https://bugs.debian.org/897790
Description: ping_host_add: Decrease buffer size to make GCC's truncation check happy.

--- a/src/liboping.c
+++ b/src/liboping.c
@@ -1605,10 +1605,8 @@
 		}
 		else
 		{
-			char errmsg[PING_ERRMSG_LEN];
-
-			snprintf (errmsg, PING_ERRMSG_LEN, "Unknown `ai_family': %i", ai_ptr->ai_family);
-			errmsg[PING_ERRMSG_LEN - 1] = '\0';
+			char errmsg[64];
+			snprintf (errmsg, sizeof(errmsg), "Unknown `ai_family': %d", ai_ptr->ai_family);
 
 			dprintf ("%s", errmsg);
 			ping_set_error (obj, "getaddrinfo", errmsg);
